cms/djangoapps/contentstore/config/models.py

```diff
@@ -0,0 +1,79 @@
+"""
+Models for configuration of the feature flags
+controlling the new assets page.
+"""
+from config_models.models import ConfigurationModel
+from django.db.models import BooleanField
+from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
+
+
+class NewAssetsPageFlag(ConfigurationModel):
+    """
+    Enables the in-development new assets page from studio-frontend.
+
+    Defaults to False platform-wide, but can be overriden via a course-specific
+    flag. The idea is that we can use this to do a gradual rollout, and remove
+    the flag entirely once generally released to everyone.
+    """
+    # this field overrides course-specific settings to enable the feature for all courses
+    enabled_for_all_courses = BooleanField(default=False)
+
+    @classmethod
+    def feature_enabled(cls, course_id=None):
+        """
+        Looks at the currently active configuration model to determine whether
+        the new assets page feature is available.
+
+        There are 2 booleans to be concerned with - enabled_for_all_courses,
+        and the implicit is_enabled(). They interact in the following ways:
+            - is_enabled: False, enabled_for_all_courses: True or False
+                - no one can use the feature.
+            - is_enabled: True, enabled_for_all_courses: False
+                - check for a CourseNewAssetsPageFlag, use that value (default False)
+                - if no course_id provided, return False
+            - is_enabled: True, enabled_for_all_courses: True
+                - everyone can use the feature
```

Comment:
The somewhat non-obvious key here is that `NewAssetsPageFlag` inherits an implicit `enabled` property (wrapped in the `is_enabled()` function), which we combine with the new `enabled_for_all_courses` value.